docs/src/main/tut/apply.md

```diff
@@ -7,7 +7,7 @@ scaladoc: "#cats.Apply"
 ---
 # Apply
 
-`Apply` extends the [`Functor`](functor.html) type class (which features the familiar `map`
+`Apply` extends the [Functor](functor.html) type class (which features the familiar `map`
```

Comment:
We just merged #459 which added the backticks to links. We should adopt one style and use that throughout the docs. What is the rationale for removing the backticks here?